zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -207,6 +217,9 @@ object Server {
 
   def requestDecompression(strict: Boolean): UServer = Server.RequestDecompression(enabled = true, strict = strict)
 
+  def responseCompression(contentSizeThreshold: Int, options: IndexedSeq[CompressionOptions]): UServer =
```

Comment:
Add `ServerConfig.ResponseCompression` and add `responseCompression: Option[ResponseCompression]` to `ServerConfig` case class, possibly choosing a reasonable default.

Reply:
Thanks for the review @jdegoes
Will add them next.
Thanks.